src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int randomRange = 101;
+
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(randomRange);
```

Comment:
Redundant, see my comment below, you should directly assign the value to ball field.